openmetadata-ui/src/main/resources/ui/src/components/Explore/ExploreTree/ExploreTree.interface.ts

```diff
@@ -48,3 +50,276 @@ export type DatabaseFields =
+  Mlflow = 'ML Flow',
+  SageMaker = 'SageMaker',
+  Sklearn = 'Sklearn',
+  VertexAI = 'Vertex AI',
+}
+enum FormattedMetadataServiceType {
+  Alation = 'Alation',
+  AlationSink = 'AlationSink',
+  Amundsen = 'Amundsen',
+  Atlas = 'Atlas',
+  MetadataES = 'MetadataES',
+  OpenMetadata = 'OpenMetadata',
+}
+
+enum FormattedPipelineServiceType {
+  Airbyte = 'Airbyte',
+  Airflow = 'Airflow',
+  CustomPipeline = 'Custom Pipeline',
+  DBTCloud = 'DBT Cloud',
+  Dagster = 'Dagster',
+  DataFactory = 'DataFactory',
+  DatabricksPipeline = 'DataBricks Pipeline',
+  DomoPipeline = 'Domo Pipeline',
+  Fivetran = 'Fivetran',
+  Flink = 'Flink',
+  GluePipeline = 'Glue Pipeline',
+  KafkaConnect = 'Kafka Connect',
+  Matillion = 'Matillion',
+  Nifi = 'Nifi',
+  OpenLineage = 'Open Lineage',
+  Spark = 'Spark',
+  Spline = 'Spline',
+  Stitch = 'Stitch',
+  Wherescape = 'Wherescape',
+}
+enum FormattedSearchServiceType {
+  CustomSearch = 'Custom Search',
+  ElasticSearch = 'Elastic Search',
+  OpenSearch = 'Open Search',
+}
+enum FormattedDatabaseServiceType {
+  Athena = 'Athena',
+  AzureSQL = 'Azure SQL',
+  BigQuery = 'Big Query',
+  BigTable = 'Big Table',
+  Cassandra = 'Cassandra',
+  Clickhouse = 'Clickhouse',
+  Cockroach = 'Cockroach',
+  Couchbase = 'Couchbase',
+  CustomDatabase = 'Custom Database',
+  Databricks = 'Databricks',
+  Datalake = 'Datalake',
+  Db2 = 'Db2',
+  Dbt = 'Dbt',
+  DeltaLake = 'DeltaLake',
+  DomoDatabase = 'Domo Database',
+  Doris = 'Doris',
+  Druid = 'Druid',
+  DynamoDB = 'Dynamo DB',
+  Exasol = 'Exasol',
+  Glue = 'Glue',
+  Greenplum = 'Greenplum',
+  Hive = 'Hive',
+  Iceberg = 'Iceberg',
+  Impala = 'Impala',
+  MariaDB = 'Maria DB',
+  MongoDB = 'Mongo DB',
+  Mssql = 'MS SQL',
+  Mysql = 'MySQL',
+  Oracle = 'Oracle',
+  PinotDB = 'PinotDB',
+  Postgres = 'Postgres',
+  Presto = 'Presto',
+  QueryLog = 'QueryLog',
+  Redshift = 'Redshift',
+  SAS = 'SAS',
+  SQLite = 'SQLite',
+  Salesforce = 'Salesforce',
+  SapERP = 'SAP ERP',
+  SapHana = 'SAP Hana',
+  SingleStore = 'SingleStore',
+  Snowflake = 'Snowflake',
+  Synapse = 'Synapse',
+  Teradata = 'Teradata',
+  Trino = 'Trino',
+  UnityCatalog = 'UnityCatalog',
+  Vertica = 'Vertica',
+}
+enum FormattedDashboardServiceType {
+  CustomDashboard = 'Custom Dashboard',
+  DomoDashboard = 'Domo Dashboard',
+  Lightdash = 'Lightdash',
+  Looker = 'Looker',
+  Metabase = 'Metabase',
+  MicroStrategy = 'Micro Strategy',
+  Mode = 'Mode',
+  PowerBI = 'PowerBI',
+  PowerBIReportServer = 'PowerBI Report Server',
+  QlikCloud = 'Qlik Cloud',
+  QlikSense = 'Qlik Sense',
+  QuickSight = 'Quick Sight',
+  Redash = 'Redash',
+  Sigma = 'Sigma',
+  Superset = 'Superset',
+  Tableau = 'Tableau',
+}
+enum FormattedMessagingServiceType {
+  CustomMessaging = 'Custom Messaging',
+  Kafka = 'Kafka',
+  Kinesis = 'Kinesis',
+  Redpanda = 'Redpanda',
+}
+
+enum FormattedAPIServiceType {
+  REST = 'Rest',
+  Webhook = 'Webhook',
+}
+
+const serviceTypeFormatters = {
```

Comment:
@dhruvjsx we need to update this each time we add new service right?
Since it's a const need to move it to a const file